### PR TITLE
Convert historian test to jest.

### PR DESCRIPTION
**Pull request by anmic**

### CODE REVIEW

**Comment by vmihailenco** [comment]
See https://github.com/facebook/jest/issues/2071 - you need to polyfil fetch / XHR etc

**Comment by anmic** [comment]
Fixed test.

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "babel-plugin-transform-define": "^1.3.1",
     "chai": "4.2.0",
     "imports-loader": "0.8.0",
+    "isomorphic-fetch": "^2.2.1",
```
Comment (vmihailenco) [inline package.json]: remove and use `cross-fetch` that is already a dependency

**Comment by anmic** [comment]
Removed 'cross-fetch' package.

**Pull request merged**